### PR TITLE
make formatter param passed by the user have higher priority

### PR DESCRIPTION
**Pull request by satancp**

### CODE REVIEW

**Comment by popomore** [comment]
ci is failed

File: lib/utils.js [hunk 1]
```diff
@@ -80,7 +80,7 @@ module.exports = {
     meta = meta || {};
     let message;
     let output;
-    const formatter = meta.formatter || options.formatter;
+    const formatter = options.formatter || meta.formatter;
```
Comment (popomore) [inline lib/utils.js]: meta should have high priority, it will be passed from function for each level.

**Comment by popomore** [comment]
I don’t know what problem do you have? you can code a testcase to show it.

**Comment by satancp** [comment]
In my case, I need modify error logs. However, when I pass a custom formatter in the config file, it does not accept it and just use a formatter named ContextFormatter which is defined by default.

**Comment by atian25** [comment]
see https://github.com/eggjs/egg-logger/pull/37#discussion_r245184817 if you want to custom context formatter workaround

**Pull request closed without merging**